pkg/ctl/delete/nodegroup.go

```diff
@@ -52,6 +57,7 @@ func deleteNodeGroupWithRunFunc(cmd *cmdutils.Cmd, runFunc func(cmd *cmdutils.Cm
 		fs.DurationVar(&maxGracePeriod, "max-grace-period", defaultMaxGracePeriod, "Maximum pods termination grace period")
 		defaultDisableEviction := false
 		fs.BoolVar(&disableEviction, "disable-eviction", defaultDisableEviction, "Force drain to use delete, even if eviction is supported. This will bypass checking PodDisruptionBudgets, use with caution.")
+		fs.IntVar(&parallel, "parallel", 5, "Number of nodes to drain in parallel. Max 25")
```

Comment:
The default should be 1. There should be no parallel draining because that changes the default behaviour of eksctl. Not to mention that the wait period between node drains will change for people who want to slow down draining of individual nodes.